### PR TITLE
VS C++ support

### PR DESCRIPTION
**Pull request by esweet431**
- Add info on Visual Studio C++ development with WSL to environment.md
- Add image of VS C++ support 
- Add C++ tutorial to TOC

### CODE REVIEW

File: WSL/toc.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@
       href: tutorials/wsl-database.md
     - name: Get started with Docker remote containers
       href: tutorials/wsl-containers.md
+    - name: Get started with Visual Studio for C++ development
+      href: /cpp-docs/build/walkthrough-build-debug-wsl2.md
```
Comment (craigloewen-msft) [inline WSL/toc.yml:30]: In the preview the link to this doc is giving me a 404 error. 
@mattwojo would you know how to properly link this?

File: WSL/setup/environment.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ For example, when storing your WSL project files:
 
 ![Windows File Explorer displaying Linux storage](../media/windows-file-explorer.png)
 
-## Set up VS Code for editing and debugging
+## Set up VS Code or Visual Studio for editing and debugging
```
[the remark this answers is not in the document]
Comment (mattwojo) [reply WSL/setup/environment.md]: Just sent a PR to update these titles accordingly and fix the relative link

File: WSL/toc.yml [hunk 3]
```diff
@@ -26,6 +26,8 @@
       href: tutorials/wsl-database.md
     - name: Get started with Docker remote containers
       href: tutorials/wsl-containers.md
+    - name: Get started with Visual Studio for C++ development
+      href: /cpp/build/walkthrough-build-debug-wsl2.md
```
Comment (craigloewen-msft) [inline WSL/toc.yml:30]:
```suggestion
      href: /cpp/build/walkthrough-build-debug-wsl2
```

**Comment by esweet431** [comment]
@craigloewen-msft @mattwojo are the relative links not supposed to end in .md? If so, I see Craig fixed the TOC, but we will also need to remove the .md from the link in environment.md

**Comment by craigloewen-msft** [comment]
Good catch @esweet431 ! The `.md` link doesn't work for me. I'll add in a fix in the `environment.md` file

**Comment by craigloewen-msft** [comment]
Fixed! Links should work now
https://github.com/MicrosoftDocs/WSL/commit/22048b8292e028c264200c15080019e04f123450

**Comment by esweet431** [comment]
Thank you @craigloewen-msft!

**Pull request merged**